typo3-fractor/rules/TYPO3v10/Yaml/TranslationFileFractor.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Typo3Fractor\TYPO3v10\Yaml;
+
+use a9f\FractorYaml\Contract\YamlFractorRule;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+final class TranslationFileFractor implements YamlFractorRule
+{
+    /**
+     * @var string
+     */
+    private const TRANSLATION_FILE_KEY = 'translationFile';
+
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Use key translationFiles instead of translationFile', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+TYPO3:
+  CMS:
+    Form:
+      prototypes:
+        standard:
+          formElementsDefinition:
+            Form:
+              renderingOptions:
+                translation:
+                  translationFile:
+                    10: 'EXT:form/Resources/Private/Language/locallang.xlf'
+                    20: 'EXT:myextension/Resources/Private/Language/locallang.xlf'
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+TYPO3:
+  CMS:
+    Form:
+      prototypes:
+        standard:
+          formElementsDefinition:
+            Form:
+              renderingOptions:
+                translation:
+                  translationFiles:
+                    20: 'EXT:myextension/Resources/Private/Language/locallang.xlf'
+CODE_SAMPLE
+            ),
+        ]);
+    }
+
+    public function refactor(array $yaml): array
```

Comment:
I think we should have a common understanding of the return values of these methods in all different file types. IMO, there's three possible return types:

- a changed (part of) the file structure
- null = no change
- some flags to signal "throw this away", "stop modifying it" etc.